rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,184 @@
+
+If an issue has been marked `tc-hold` for more than two weeks with no response or acknowledgement from the TC, the
+developer SHOULD post to the Internals mailing list a notification of absence.
+
+If a notification of absence receives no response from the TC or any member of it for one month, then the entire TC
+is considered AWOL and removed.  New elections must then be immediately called as described above.
+
+The election of a new TC will "reset the clock" on any outstanding `tc-hold` issues.
+
+==== Decision process ====
+
+=== User-facing changes ===
+
+The RFC process will continue to be used for user facing changes.  However, it will no longer be used for
+non-user-facing technical decisions.
+
+TC members have no special role or authority in the RFC process.
+
+=== Process changes ===
+
+The RFC process will continue to be used for process changes, including changes to the role of the TC.
+
+=== Technical changes ===
+
+As noted, the TC has no special role or authority on user-facing changes.  The TC may, however, decide on the
+implementation of those features. In other words the TC can decide about implementation and ABI but not about API.
+
+The TC SHOULD NOT block the merging of a user-facing change approved by the RFC process, unless the provided
+implementation would result in introduction of new bugs, side effects not mentioned in the RFC, significant performance
+penalties, or if there is an equivalent implementation in progress that TC finds more appropriate.
+
+The TC SHOULD give due consideration and weight to previous decisions regarding technical aspects, either by the TC
+or by RFCs prior to the TC's creation.  However, it is NOT REQUIRED to abide by them, and may make decisions as it
+sees best to ensure the quality and maintainability of the code at the time the decision is made.
+
+Technical aspects on which the TC may make a decision include, but are not limited to:
+
+  * Whether a given PR is a feature or a bug.
+  * Whether a given change qualifies as user-facing, and thus requires an RFC.
+  * Whether a given technical aspect or implementation approach is acceptable.
+
+=== TC voting ===
+
+Once an issue has been brought to the TC, the Secretary must allow all TC members time to offer their input within one
+week.  Once the Secretary feels the issue has been sufficiently discussed, the Secretary will call a vote of the TC
+members. The vote may be public or private and conducted however the TC chooses.
+
+The vote will last for one week or until all TC members have voted, whichever comes first.  All votes are
+simple-majority, with no quorum on a binary question (which could be yes/no or either/or, depending on the situation,
+as determined by the Secretary).  Members may explicitly abstain, and refraining from voting counts as an abstention.
+Abstentions do not count toward the result.
+
+In case of a tie, the Secretary's vote will be considered the winning position.
```

Comment:
This means, in case of 4 members (including the secretary) the secretary essentially has no vote unless one voter abstains. Is that acceptable?

Reply:
No, the secretary is a member of TC so it has the same vote as other members. It is actually oposite because being a secretary, there is an additional vote for a secretary in case of a tie. This is kind of an implication as we don't define anywhere that secretary looses its vote by becoming a secretary. Do you think this needs extra clarification?